src/main/webapp/app/core/landing/landing.component.html

```
@@ -1,9 +1,14 @@
+<a class="visually-hidden-focusable" href="#main">Skip to main content</a>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)